spec/webmock_helper.rb

```diff
@@ -0,0 +1,12 @@
+require 'webmock/rspec'
+require 'support/europeana_api_helper'
+require 'support/europeana_blog_helper'
+
+RSpec.configure do |config|
+  config.before(:each) do
+    ENV['EUROPEANA_API_KEY'] = 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.